app/src/main/java/com/github/palFinderTeam/palfinder/ProfileActivity.kt

```diff
@@ -126,6 +136,26 @@ class ProfileActivity : AppCompatActivity() {
         desc.maxLines = Integer.MAX_VALUE
     }
 
+    /**
+     * Clicking on QR Code icon will show QR code
+     */
+    fun showQR(view: View?) {
+        //Initiate the barcode encoder
+        val barcodeEncoder = BarcodeEncoder()
+        //Encode text in editText into QRCode image into the specified size using barcodeEncoder
+        val bitmap = barcodeEncoder.encodeBitmap(intent.getStringExtra(USER_ID), BarcodeFormat.QR_CODE, 512, 512)
```

Comment:
This number could be written somewhere maybe in values.xml, string, or as a companion object

Reply:
Done :)